opentelekomcloud/provider.go

```diff
@@ -460,6 +461,7 @@ func Provider() *schema.Provider {
 			"opentelekomcloud_fw_policy_v2":                              fw.ResourceFWPolicyV2(),
 			"opentelekomcloud_fw_rule_v2":                                fw.ResourceFWRuleV2(),
 			"opentelekomcloud_gaussdb_mysql_instance_v3":                 gaussdb.ResourceGaussDBInstanceV3(),
+			"opentelekomcloud_hss_host_group_v5":                         hss.ResourceHostGroup(),
```

Comment:
Because in documentation it v5 and for future reasons we trying to keep versions: https://docs.otc.t-systems.com/host-security-service/api-ref/api_description/server_management/querying_ecss.html#listhoststatus